app/api/controllers/validators/resolvers/ResolveFromAndToDates.scala

```diff
@@ -34,4 +34,14 @@ object ResolveFromAndToDates extends ResolverSupport {
   val resolver: Resolver[(String, String), DateRange] = resolveDateRange.withYearsLimitedTo(minimumYear, maximumYear)
 
   def apply(value: (String, String)): Validated[Seq[MtdError], DateRange] = resolver(value)
+
+  def apply(value: (Option[String], Option[String])): Option[Validated[Seq[MtdError], DateRange]] = {
```

Comment:
This will only validate the dates if both dates are supplied. Not sure if this is what we want.